src/main/java/com/epam/controller/IndexController.java

```diff
@@ -1,19 +1,34 @@
 package com.epam.controller;
 
+import com.epam.model.User;
+import com.epam.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 
-import java.time.LocalDateTime;
-import java.util.HashMap;
-import java.util.Map;
 
 @Controller
+@RequestMapping(path="/demo")
 public class IndexController {
-    @GetMapping("/")
-    public ModelAndView index() {
-        Map<String, String> model = new HashMap<>();
-        model.put("name", "Alexey");
-        return new ModelAndView("index", model);
+    @Autowired
+    private UserRepository userRepository;
+
+    @GetMapping(path="/add")
+    public @ResponseBody String addNewUser (@RequestParam String name) {
+
+
+        User n = new User();
+        n.setFirstName(name);
+        userRepository.save(n);
+        return "Saved";
+    }
+
+    @GetMapping(path="/all")
```

Comment:
That's rather ambiguous url for REST... You'll be out of URLs pretty fast if you'll keep following such naming convention (like, what url you'll use for getting all rooms, /all2 ?). In a RESTful application your URL should describe what kind of resource you're accessing - for example if you're trying to get a list of all rooms URL should be '/rooms' and its the same for users, meals etc. You can start with something like [this article](https://restfulapi.net/resource-naming/) to get a better understanding of the subject.